text/0000-base-image-label.md

```diff
@@ -0,0 +1,73 @@
+# Meta
+[meta]: #meta
+- Name: Add base image SHA to stack metadata
+- Start Date: 2020-10-23)
+- Author(s): dumez-k
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+Building off of this earlier [RFC](https://github.com/buildpacks/rfcs/blob/main/text/0050-stack-metadata.md) outlining stack metadata,
+we would like to add a label to expose the stack base image SHA in the metadata of all stack-cnb images.
+
+# Motivation
+[motivation]: #motivation
+
+- Allows stack users and authors to easily see which base-image that each stack cnb image is based off of.
+
+# What it is
+[what-it-is]: #what-it-is
+
+This provides a high level overview of the feature.
+
+- Define the target persona: stack user, stack author
+
+The label can be **optionally** added by stack maintainers to stack image:
+ 
+`io.buildpacks.stack.base_image:` SHA of stack base image
+
+
+# How it Works
+[how-it-works]: #how-it-works
+
+This is the technical portion of the RFC, where you explain the design in sufficient detail.
+
+The `base_image` label will be added as a `Label` in the stack's Dockerfile.
+Example:
+
+`LABEL io.buildpacks.stack.base_image=e3c1c2659c2468ea8ac8058c501293882b0a55622840eee53525f90b59d31a33`
```

Comment:
Without the registry and repository, this doesn't seem like enough information for a user to actually find the image.

Reply:
yeah this is definitely a good point, we should change this to the fully qualified path